apps/meteor/app/utils/lib/getValidRoomName.js

```diff
@@ -2,15 +2,14 @@ import { Meteor } from 'meteor/meteor';
 import limax from 'limax';
 import { escapeHTML } from '@rocket.chat/string-helpers';
 
-import { settings } from '../../settings';
-import { Rooms } from '../../models';
+import { Settings, Rooms } from '../../models';
 import { validateName } from '../../lib/server/functions/validateName';
 
 export const getValidRoomName = (displayName, rid = '', options = {}) => {
 	let slugifiedName = displayName;
 
-	if (settings.get('UI_Allow_room_names_with_special_chars')) {
-		const cleanName = limax(displayName);
+	if (Settings.findOneById('UI_Allow_room_names_with_special_chars').value) {
```

Comment:
Since we're not changing the value of this setting before starting the importing process (due to the changes you've made in the `ImporterBase.js` file), I guess there's no problem on using the cached settings here. wdyt? Maybe we could keep using the `settings` import in this file

Reply:
I agree with you @matheusbsilva137